### DISCUSSION TITLE
Proton version for modding

### DISCUSSION THREAD

**Discussion by alicx1** [thijsvanloef/palworld-server-docker#514] [Ideas]
Hey,
I would just suggest to offer the possibility to have a server running with proton so that we can install UE4SS on it and enjoy modding.

**Comment by thijsvanloef** [comment]
Hi @alicx1 If you want, you can open a [Feature request](https://github.com/thijsvanloef/palworld-server-docker/issues/new?assignees=&labels=enhancement&projects=&template=feature_request.md&title=) to have this feature implemented. Or if you are willing and able to implement this feature yourself and open a Pull Request.

**Reply by alicx1** [reply]
Hi @thijsvanloef  I'll wait for the next palworld update then create a feature request. if i have some time i could give it a try too.
